S2VX.Game.Tests/HeadlessTests/EditorHoldNoteTests.cs

```diff
@@ -38,6 +39,10 @@ public class EditorHoldNoteTests : S2VXTestScene {
                 HitTime = Story.Notes.ShowTime + Story.Notes.FadeInTime + NoteAppearTime,
                 EndTime = Story.Notes.ShowTime + Story.Notes.FadeInTime + NoteAppearTime + HoldDuration
             }));
+            AddStep("Set max note alpha to 1", () => Story.AddCommand(new HoldNotesAlphaCommand {
+                StartValue = 1.0f,
+                EndValue = 1.0f
+            }));
```

Comment:
Why do we need these?

Reply:
The default alpha for HoldNotes has been changed to 0.7 in HoldNotesAlphaCommand.cs
Explicitly setting the alpha to 1.0 in SetUpSteps is now needed to keep the test expectations consistent. I figured it was better than hard coding all the expectations to 0.7 since we may change that default again.